pywinauto/base_wrapper.py

```diff
@@ -517,7 +518,7 @@ def is_child(self, parent):
         element if the parent element is the the chain of parent elements
         for the child element.
         """
-        return self in parent.children()
+        return self in parent.children(class_name = self.class_name())
```

Comment:
Good! This should really boost the method. Not so obviously first, but I got it.